### PR TITLE
add file sort option in cdc case

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

Support test file sort in CDC

### What is changed and how does it work?

- Add two options, `cdc_sort-engine` and `cdc_sort-dir`
- Add one cron workflow template `argo/cron/cdc-file-sort.yaml` (will this cron be deployed in ticpket automatically?)

### Check List 

Tests 

 - Unit test
 - E2E test

### Does this PR introduce a user-facing change?:

- release-note
    NONE

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @mahjonp

**Comment by mahjonp** [comment]
I can help you to apply this workflow `argo/cron/cdc-file-sort.yaml`

File: argo/cron/cdc-file-sort.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+metadata:
+  name: tipocket-cdc
```
Comment (mahjonp) [inline argo/cron/cdc-file-sort.yaml:2]: We should rename `tipocket-cdc`

Comment (amyangfei) [reply argo/cron/cdc-file-sort.yaml:2]: oh, yes

Comment (amyangfei) [reply argo/cron/cdc-file-sort.yaml:2]: fixed in https://github.com/pingcap/tipocket/pull/298

**Pull request merged**